docs/test/using-microsoft-visualstudio-testtools-unittesting-members-in-unit-tests.md

```diff
@@ -239,7 +239,7 @@ public class MyOtherTestClass
 
 It's possible to control the inheritance behavior: only for current class using `InheritanceBehavior.None` or for all derived classes using `InheritanceBehavior.BeforeEachDerivedClass`.
 
-It's also possible to configure whether the class cleanup should be run at the end of the class or at the end of the assembly (current default).
+It's also possible to configure whether the class cleanup should be run at the end of the class or at the end of the assembly (No longer supported starting from MSTest v4 as EndOfClass is the default and only ClassCleanup behavior).
```

Comment:
Same as other PR. Can we make "ClassCleanup" a link? I don't think it's mentioned elsewhere in this doc, so might be confusing.